fs/dafsserver/dafsserver.cpp

```diff
@@ -1882,6 +1884,11 @@ class CRemoteXmlReadActivity : public CRemoteMarkupReadActivity
 public:
     CRemoteXmlReadActivity(IPropertyTree &config, IFileDescriptor *fileDesc) : PARENT(config, fileDesc, TAKxmlread)
     {
+        xpath.set("/Dataset/");
+        if (customRowTag.isEmpty()) // no override
+            fileDesc->queryProperties().getProp("@rowTag", xpath);
```

Comment:
does getProp append on to xpath? if it performs a set(), then we have slightly different behavior if we use this.customRowTag, and properties[@rowTag]

Reply:
@rpastrana IPropertyTree::getProp (and other get function) append to the output StringBuffer.